typescript/src/util.ts

```diff
@@ -105,3 +105,10 @@ String.prototype.toSnakeCase = function(this: string): string {
   }
   return words.join('_');
 };
+
+Array.prototype.addStringType = function(this: string[]): string[] {
+  if (this.length === 0) {
+    return this;
+  }
+  return this.map(p => p + ': string');
```

Comment:
Here we basically incorporate a part of TypeScript output into the source code (and not the templates). We are trying not to do it that often. (the generator code actually has nothing to do with TypeScript output, and it's theoretically possible to generate any other language by changing just the templates)

Can we do template iteration with a `joiner()` instead?

Reply:
Sure thing, great suggestion!